src/config/graphical-session/graphics-drivers/nvidia.md

```diff
@@ -0,0 +1,86 @@
+# NVIDIA
+
+## nouveau (Open Source Driver)
+
+This driver is developed mostly by the community, with little input from Nvidia,
+and is not as performant as the proprietary driver. It is required in order to
+run most Wayland compositors.
+
+Install the Mesa DRI driver `mesa-dri` or the Xorg driver `xf86-video-nouveau`.
```

Comment:
`mesa-dri` is a package, not a driver; likewise for `xf86-video-nouveau`. Both *provide* drivers.

i think it would be helpful to specify the names of the drivers provided by each package.